engines/director/archive.cpp

```diff
@@ -1148,4 +1168,214 @@ Common::String RIFXArchive::formatArchiveInfo() {
 	return result;
 }
 
+bool RIFXArchive::writeToFile(Common::Path path) {
+	// The offsets used in this function should be not the same as original
+	// That defeats the whole purpose of writing movies
+
+	// ignoring the startOffset
+	// For RIFX stream, moreoffset = 0
+	byte *dumpData = nullptr;
+
+	// Don't need to allocate this much size in case 'JUNK' and 'FREE' resources are ignored
```

Comment:
```suggestion
	// Don't need to allocate this much size in case 'junk' and 'free' resources are ignored
```
Those resources are named with all lowercase letters